.travis.yml

```diff
@@ -1,15 +1,17 @@
 language: php
 
 php:
-  - 5.6
-  - 7.0
   - 7.1
+  - 7.2
```

Comment:
We can also start to test against PHP 7.3, Travis has already provide the images